pkg/pillar/cmd/tpmmgr/tpmmgr.go

```diff
@@ -848,6 +850,73 @@ func testEcdhAES() error {
 	return nil
 }
 
+// DecryptWithCipherInfo : Decryption API, for encrypted object information received from controller
+func DecryptWithCipherInfo(cipherBlock types.CipherBlock) (string, error) {
```

Comment:
Will this always return a (ASCII or UTF-8) string, or is it actually returning bytes?